README.md

```diff
@@ -4,12 +4,13 @@
 
 > Description the project.
 
+This repository contains an web app that allows to add and remove book's data using Javascript
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Javascript
+- HTML
+- CSS
 
 ## Live Demo (if available)
```

Comment:
- [ ] This section is not needed for this particular project milestone, and the link does not provide any live demo.  Kindly delete this section.